lib/stringify.js

```diff
@@ -142,7 +142,7 @@ var stringify = function stringify(
     var objKeys;
     if (generateArrayPrefix === 'comma' && isArray(obj)) {
         // we need to join elements in
-        objKeys = [{ value: obj.length > 0 ? obj.join(',') || null : undefined }];
+        objKeys = [{ value: obj.length > 0 ? obj.join(',') || null : (void 0) }];
```

Comment:
```suggestion
        objKeys = [{ value: obj.length > 0 ? obj.join(',') || null : void undefined }];
```
since it's a bit clearer